buildpack.toml

```diff
@@ -1,4 +1,8 @@
+api = "0.2"
+
 [buildpack]
+id = "heroku/jvm"
+version = "0.1"
 name = "JVM"
 
   [publish.Ignore]
```

Comment:
in the long term, I wonder if having these extra keys that don't comply with `buildpack.toml` in v3 matter.

Reply:
we could have buildpack registry look for them in the `[metadata]`